sympy/physics/vector/vector.py

```diff
@@ -695,6 +695,19 @@ def applyfunc(self, f):
             d[v[1]] = v[0].applyfunc(f)
         return Vector(d)
 
+    def free_symbols(self, reference_frame):
+        """
+        Returns from the vector those which are free symbols.
```

Comment:
"Returns the free symbols in the measure numbers of the vector expressed in the given reference frame."

Reply:
@moorepants apologies for such silly mistakes. I have revised the function free_symbols, please check:

    def free_symbols(self, reference_frame):
        """
        Returns the free symbols in the measure numbers of the vector
        expressed in the given reference frame.

        Parameter
        =========

        reference_frame : ReferenceFrame
            The frame with respect to which the free symbols of the
            given vector is to be determined.

        """

        return self.to_matrix(reference_frame).free_symbols

Reply:
Looks good. These are not mistakes or silly. I'm just showing you some of the conventions we use in SymPy to keep the code and doc quality high.